go/pkg/chunker/chunker.go

```diff
@@ -21,6 +22,9 @@ var IOBufferSize = 10 * 1024 * 1024
 // ErrEOF is returned when Next is called when HasNext is false.
 var ErrEOF = errors.New("ErrEOF")
 
+// Compressor for full blobs
+var fullCompressor, _ = zstd.NewWriter(nil)
```

Comment:
Is this one thread safe?

Reply:
`Encoder.EncodeAll` is thread-safe, and that's all that is used below.

Reply:
@mostynb is right here - I expanded the comments :)